File: week2/scores.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int size = get_int("How many scores you want to average? ");
    int scores[size];
    int sum = 0;

    for (int i = 0; i < size; i++)
    {
        scores[i] = get_int("Score%i: ", (i + 1));
        sum += scores[i];
    }
    printf("Average : %.2f\n", sum / (float) size);
}

File: week6/scores.py
# take scores from user input and calculate average
from cs50 import get_int

scores = []
n = get_int("Number of scores to add: ")

for i in range(n):
    score = get_int("score: ")
    # appends the score to end of list
    scores.append(score)
    # scores = scores + [score]

average = sum(scores) / len(scores)
print(f"Average: {average}")
